docs/exit_validation.md

```diff
@@ -52,7 +52,8 @@ Causes to emit an `:invalid_finalization` event
     * if `true` -> noop,
     * if `false` -> emit `:invalid_exit` event  which leads to challenge
     * if `false` and there is less than `sla_margin` time till finalization -> `:unchallenged_exit`
-4. Spend utxos in `State` on exit finalization or challenging
+4. Spend utxos in `State` on exit finalization or challenging by call of `ExitProcessor.check_validity` method 
```

Comment:
this is not correct I think. Weren't we supposed instead to add a point here and say how validity checks are done, when and by what?

Reply:
```
....
4. Spend utxos in `State` on exit finalization or challenging
....
6. `ExitProcessor` check validation of exits is call periodically by itself with `exit_processor_validation_interval_ms`
interval and by `BlockGetter` every time when applying new block. 
```